scripts/valhalla_build_config

```diff
@@ -512,6 +513,7 @@ help_text = {
         'max_reachability': 'Maximum reachability (number of nodes reachable) allowed on any one location',
         'max_radius': 'Maximum radius in meters allowed on any one location',
         'max_timedep_distance': 'Maximum b-line distance between locations to allow a time-dependent route',
+        'max_timedep_distance_matrix': 'Maximum b-line distance between 2 most distant locations in meters to allow a time-dependent matrix',
```

Comment:
if this is exceeded for a source or target, the whole expansion of that source or target won't track time